### DISCUSSION TITLE
several suggestions for better usage of Lists

### DISCUSSION THREAD

**Discussion by ljm12914** [community/community#8650] [Other Feature Feedback, Questions, & Ideas]
1. could I filter my stars by lists?
2. now the main "your stars" page is still full of repositories. consider moving listed repositories out of the main page and leave the unlisted only?
3. I want to see the added lists directly from the main page, not clicking all the time to open the menu and check.
4. this button is a little difficult to use due to its small area on screen, especially if there are lots of lists. Consider replacing the scrollbar by extending the whole menu?
![image](https://user-images.githubusercontent.com/29831474/145551414-85c9aa8d-1abe-4d03-9b19-2811ba001ebb.png)

**Comment by brianlovin** [comment]
Thank you for these suggestions @ljm12914!

1. Good question. Technically, if you were to click on an individual list, it would be functionally the same as viewing a filtered set of all your starred repos. Does that make sense for your use case?
2. I'll add your +1 to our tracking issue for this — several other people have requested something similar, it sounds like an interesting idea for us.
3. Agreed, this is not ideal right now. I'll add your feedback to our issue for this!
4. I'm not quite sure which button you're referring to for this one — are you referring to the button that triggers the list selection dropdown?
